src/docs/manual/index.html

```diff
@@ -975,6 +975,9 @@ <h2 id="command-line-options">Command-line options</h2>
  <p>Using <code>--methodlist</code> is less common, and more error-prone, than <code>--testjar</code>,
  <code>--test-package</code>, <code>--classlist</code>, or <code>--testclass</code>. See the notes about <a href="https://randoop.github.io/randoop/manual/#specifying-methods">specifying methods that may
  appear in a test</a>.
+            <li id="option:use-reflection"><b>--use-reflection=</b><i>boolean</i>.
```

Comment:
This pull request adds a command-line option `--use-reflection` that is adjacent to `--omit-classes`, but I think it should be next to `--only-test-public-members` which it is more similar to.  Also, let's improve the name `--use-reflection` which is an implementation detail.  The real question is whether private, protected, and or package-private methods should be called.
So maybe there should be an enum indicating how many methods should be invoked:
 1. public only
 2. public + some package-private (based on `--junit-package-name`)
 3. public + package-private
 4. public + package-private + protected
 5. public + package-private + protected + private

The current Randoop code supports #1 and #2.
Your code adds #5, though I don't think that `--only-test-public-members` is respected if `--use-reflection` is passed.  Having a single command-line argument will prevent that kind of inconsistency.  (We will keep the `--only-test-public-members` command-line argument for a while, for backward compatibility.)

It's true that for protected and private methods (and some package-private ones), they will have to be called reflectively.

In MethodCall, I've renamed the `isAccessible` boolean field to `callReflectively`, which I think is what it is really about.  I think that will make the code clearer.

Another way of saying the above is that the current code conflates "accessibility" with *whether* a method should be called.  They used to be the same, before your extended functionality, but now they should be separated for clarity:
 * which methods should be called.
 * whether a method must be called reflectively.